users/vieting/experiments/switchboard/ctc/feat/default_tools.py

```diff
@@ -31,6 +31,6 @@
 
 RETURNN_ROOT = CloneGitRepositoryJob(
     "https://github.com/rwth-i6/returnn",
-    commit="6d2945a85cc95df5349a59541d84f172dd55cc20",
```

Comment:
Updated to a newer Returnn commit. Was needed to solve some older Problems. 
Maybe change this part completely to not use fixed commits, but always the newest commit?

Reply:
I think it's better to have the hash in there. Also, you might change it for only a subset of the experiments if it is only needed there to make sure everything is reproducible.

Reply:
reverted to old hash